lib/response/sfWebResponse.class.php

```diff
@@ -359,7 +359,8 @@ public function sendHttpHeaders()
         // cookies
         foreach ($this->cookies as $cookie) {
             $expire = isset($cookie['expire']) ? $cookie['expire'] : 0;
-            setrawcookie($cookie['name'], $cookie['value'], $expire, $cookie['path'], $cookie['domain'], $cookie['secure'], $cookie['httpOnly']);
+            $domain = isset($cookie['domain']) ? $cookie['domain'] : '';
+            setrawcookie($cookie['name'], $cookie['value'], $expire, $cookie['path'], $domain, $cookie['secure'], $cookie['httpOnly']);
```

Comment:
Are the other keys are always present?

Reply:
Not sure, hope other users will report issues for other cases :)
Ok to merge? wdyt?

Reply:
Sure, it's already an improvement :)